examples/timeseries/timeseries_anomaly_detection.py

```diff
@@ -126,7 +126,7 @@ def create_sequences(values, time_steps=TIME_STEPS):
 
 model = keras.Sequential(
     [
-        layers.Input(shape=(x_train.shape[1], x_train.shape[2])),
+        layers.InputLayer(input_shape=(x_train.shape[1], x_train.shape[2])),
```

Comment:
We've removed this warning (this will be live in TF 2.6). Either approach is fine now.